SMBPositionAnalytics.md

```diff
@@ -0,0 +1,136 @@
+SMB Position Analytics Service
+==============================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Carotene With Location Request](#carotene-with-location-request)
+- [Carotene Response Structure](#carotene-response-structure)
+- [Job Request](#job-request)
+- [Job Response Structure](#job-response)
+- [Versioning](#versioning)
+
+## Summary
+
+The Small Medium Business(SMB) Position Analytics service provides hiring data pertaining to job titles and currently open positions available on CareeBuilder job boards. The service has two endpoints:
+
+https://api.careerbuilder.com/core/positionanalytics/carotene
+
+https://api.careerbuilder.com/core/positionanalytics/job
+
+Requests can be sent in the form of HTTP GET or POST (form or JSON)
+
+The data returned will contain an object or group of objects representing actions taken on a carotene title or job respectively. Each action will have associated `current` and `delta` values. The `current` value will account the total number an action was performed over the last day; the `delta` will identify the change in percentage of the `current` value from the day before. Positive values in the `delta` identify an increase day over day and negative values identify a decrease.
+
+## Carotene With Location Request
+A request to the carotene endpoint provides data on how a specified job title performs in a given location.
+
+E.g. I'm a recruiter and I'm about to post a Software Engineer job in Atlanta. How many applies have their been to Software Engineer titles in this city in the last day? What is the change from the day before? On average how long does it take to fill a Software Engineer role in this location? This endpoint serves to answer these questions.
```

Comment:
I think there's a small typo here: 
> How many applies have their been ...

Shouldn't it be "there" instead of "their"?